lib/EntryManager/ConnectedWrapper.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { useOkapiKy } from '@folio/stripes/core';
```

Comment:
@MikeTaylor given this a stripes module I think we need to use `@folio/stripes-core` here

Reply:
Yep, I just landed on that! :-)

Reply:
It was realising that even the lint phase failed that gave it away!